packages/x-tailwindcss/src/x-tailwind-plugin/components/tag/default.ts

```diff
@@ -0,0 +1,59 @@
+import { TailwindHelpers } from '../../../types';
+import { tagSizes } from './sizes';
+
+/**
+ * Returns the default styles for component `tag`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+export function tagDefault(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+
+  return {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    alignContent: 'center',
+    flexFlow: 'row nowrap',
+    boxSizing: 'border-box',
+
+    backgroundColor: theme('colors.neutral.0'),
+    borderColor: theme('colors.neutral.25'),
+    borderStyle: 'solid',
+    borderWidth: theme('borderWidth.1'),
+    color: theme('colors.neutral.75'),
+
+    gap: theme('spacing.8'),
+
+    fontFamily: theme('fontFamily.main'),
+    fontWeight: theme('fontWeight.regular'),
+    letterSpacing: theme('letterSpacing.lg'),
+    lineHeight: theme('lineHeight.sm'),
+
+    cursor: 'pointer',
+
+    '&:hover,&:focus,&:active': {
+      borderColor: theme('colors.neutral.50'),
+      color: theme('colors.neutral.90')
+    },
+
+    '&:disabled': {
+      cursor: 'not-allowed',
+      borderColor: theme('colors.neutral.25'),
```

Comment:
```suggestion
```

its already in the default one

Reply:
Ok, i found that this is necessary because the :hover variant will override this value.